configure.ac

```diff
@@ -28,6 +28,7 @@ uname -mrsv 2>/dev/null
 AM_INIT_AUTOMAKE([foreign subdir-objects tar-ustar])
 AM_SILENT_RULES([yes])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIR([m4])
```

Comment:
[doc](https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/Input.html)

here we used [ax_prog_cc_for_build.m4](https://www.gnu.org/software/autoconf-archive/ax_prog_cxx_for_build.html)